servers/account-data-deleter/src/config/index.ts

```diff
@@ -70,12 +86,13 @@ export const config = {
     queryLimit: 300,
     tableNames: tables,
   },
-  sqs: {
-    accountDeleteQueue: {
-      url:
-        process.env.SQS_BATCH_DELETE_QUEUE_URL ||
-        'http://localhost:4566/000000000000/pocket-list-delete-queue',
-    },
+  listExport: {
+    exportBucket:
+      process.env.LIST_EXPORT_PARTS_BUCKET || 'com.getpocket.list-exports',
+    partsPrefix: process.env.LIST_EXPORT_PARTS_PREFIX || '',
+    archivePrefix: process.env.LIST_EXPORT_ARCHIVE_PREFIX || '',
+    queryLimit: 10000,
+    signedUrlExpiry: 60 * 60 * 24 * 7, // 7 days in seconds
```

Comment:
we should prob add 7 days to the email we send so they know

Reply:
I was thinking that as well